app/helpers/contentful_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Helper module for contentful
+# see also the contentful gem: https://github.com/contentful/contentful.rb
+module ContentfulHelper
+  def contentful
+    @client ||= Contentful::Client.new(
```

Comment:
Let's name the `@client` instance variable to be more specific, .e.g. `@contentful_client`.